modules/azure/api_management_permissions/main.tf

```diff
@@ -16,13 +16,12 @@ provider "azurerm" {
 }
 
 resource "azurerm_api_management_group" "group" {
-  // FIXME: Deze folder heeft geen `variables.tf` dus `var.groups` bestaat helemaal niet
```

Comment:
To be sure: it does exist now? As you are using vars

Reply:
I forgot to remove the fixme comment when i fixed this, has a variables.tf now